e2e/e2etests/test_extract_bitcoin_inscription_memo.go

```diff
@@ -0,0 +1,60 @@
+package e2etests
+
+import (
+	"encoding/hex"
+	"github.com/btcsuite/btcd/btcjson"
+	"github.com/rs/zerolog/log"
+	"github.com/stretchr/testify/require"
+	zetabitcoin "github.com/zeta-chain/zetacore/zetaclient/chains/bitcoin"
+	btcobserver "github.com/zeta-chain/zetacore/zetaclient/chains/bitcoin/observer"
+
+	"github.com/zeta-chain/zetacore/e2e/runner"
+)
+
+func TestExtractBitcoinInscriptionMemo(r *runner.E2ERunner, args []string) {
+	r.Logger.Print("Testing extract memo from btc inscription")
+
+	r.SetBtcAddress(r.Name, false)
+
+	// obtain some initial fund
+	stop := r.MineBlocksIfLocalBitcoin()
+	defer stop()
+	r.Logger.Print("Mined blocks")
+
+	// list deployer utxos
+	utxos, err := r.ListDeployerUTXOs()
+	require.NoError(r, err)
+
+	amount := parseFloat(r, args[0])
+	memo, _ := hex.DecodeString(
+		"72f080c854647755d0d9e6f6821f6931f855b9acffd53d87433395672756d58822fd143360762109ab898626556b1c3b8d3096d2361f1297df4a41c1b429471a9aa2fc9be5f27c13b3863d6ac269e4b587d8389f8fd9649859935b0d48dea88cdb40f20c",
```

Comment:
I think this test should call some EVM contract on zeta with long memo and then perform a check with a getter. Proposal:

1) send "lorem ipsum ....." (1024+ bytes) from BTC to zeta's contract that should store it in the map of `map[zeta_evm_address]text`
2) call `get_test_by_address(address)` getter

Reply:
https://github.com/zeta-chain/node/issues/2900